scripts/release.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright (c) The Diem Core Contributors
+# Copyright (c) The Move Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+rm -rf move-artifacts/*
+mkdir -p move-artifacts/
+BINS="move move-analyzer move-bytecode-viewer move-disassembler move-prover move-to-yul prover-lab"
+
+for BIN in $BINS; do
+  cp -v target/release/$BIN move-artifacts/
+done
+
+cp -v README.md move-artifacts/
+
+if [ "$1" == "windows-latest" ]; then
+  7z a -r move-$1.zip move-artifacts
+else
+  zip -r move-$1.zip move-artifacts
+fi
```

Comment:
This script doesn't actually do any of the building of the assets which feels a little bit odd. I'd probably add a `cargo build --release --bin <bin> --bin <bin>` to this script and then remove the build from the CI job. That way you can actually run this script without assuming that someone already did the `cargo build` step previously.

Reply:
I'll change the name of the script and add some code comments